en/drafts/originals/designing-a-timeline-tabletop-simulator.md

```diff
@@ -542,76 +534,71 @@ If we look at the script, [`PH_nandeck_Your_Timeline_TTS_Face.txt`](https://gith
+When you first open Tabletop Simulator, it asks you whether you want to join an existing game, or _Create_ a new game: choose _Create_. You will be prompted to load a Classic Game, a Digitally Licensed Game, a game from the Steam Workshop, or to _Save and Load_ your own local content. Choose _Save and Load_ and exit out of the screen. 
 
-This should leave you staring at an empty Tabletop. From the top menu options, select 'Objects', then 'Components,' and then 'Cards.' From the options presented, select 'Custom Deck.' This will add a blank deck on your virtual tabletop and you will be given the option to Import your files from _nanDECK_ to create your custom deck (if you don't see this option, right-click on the blank deck). 
+This should leave you in front of an empty tabletop. From the top menu options, select **Objects**, then **Components**, and then **Cards**. From the options presented, select **Custom Deck**. This will add a blank deck to your virtual tabletop and you will be given the option to import your files from nanDECK to create your own custom deck (if you don't see this option, right-click on the blank deck). 
 
-From this menu, you will be able to select the image of your deck face from your local drive. Make sure to check the box beside the option, 'Unique backs' and select the image of your deck back file from your computer. Leave the width option at 10, the height option at 7, but do adjust the slider so that it displays the true number of cards in your deck. After these steps are complete, hit the 'Import' button and your deck will be built for your virtual play.
+From this menu, you will be able to select the deck's face from your local drive. Make sure to check the box beside the option **Unique backs**. Leave the width option on 10 and the height option on 7, but do adjust the slider so that it displays the true number of cards in your deck. Once these steps are complete, hit the _Import_ button and your deck will be built for virtual play.
 
-By uploading your deck into TTS, you will be able to play your version of _Timeline_, with up to nine other invited players in an onine environment. Furthermore, if you choose to upload and make your game assets available through the [_Tabletop Simulator's Steam Workshop_](https://steamcommunity.com/app/286160/workshop/), you can add your deck to the over 11,000 sets of cards available to a community of over 2 million TTS customers to play (Bezerk Games). 
+You will now be able to play your version of _Timeline_ in an online environment with up to nine other invited players. Furthermore, if you choose to upload and make your game assets available through the [_Tabletop Simulator's Steam Workshop_](https://steamcommunity.com/app/286160/workshop/), you can add your deck to a collection of over 11,000 sets of cards that are available to a community of over 2 million TTS customers.[^18] 
```

Comment:
Perhaps I am missing something here, but I couldn't see where you explained how (or when) to upload the deck's back, or especially how to align it with the faces we just uploaded. I'm sorry if I'm not understanding the lesson here! I just wanted to check.

Reply:
This is explained in paragraph 155. If you think a figure might be useful to, I made this screenshot:

![image](https://github.com/programminghistorian/ph-submissions/assets/879434/7346414d-c7f4-4071-9b2d-1979e5677777)

Reply:
Thanks for confirming. I think the paragraph describes it well enough without needing an image!